assessments/projects/linkerd/self-assessment.md

```diff
@@ -0,0 +1,321 @@
+# Self-assessment
+The Self-assessment is the initial document for projects to begin thinking about the
+security of the project, determining gaps in their security, and preparing any security
+documentation for their users. This document is ideal for projects currently in the
+CNCF **sandbox** as well as projects that are looking to receive a joint assessment and
+currently in CNCF **incubation**.
+
+
+For a detailed guide with step-by-step discussion and examples, check out the free
+Express Learning course provided by Linux Foundation Training & Certification:
+[Security Assessments for Open Source Projects](https://training.linuxfoundation.org/express-learning/security-self-assessments-for-open-source-projects-lfel1005/).
+
+
+# Self-assessment outline
+
+
+## Table of contents
+
+
+* [Metadata](#metadata)
+ * [Security links](#security-links)
+* [Overview](#overview)
+ * [Actors](#actors)
+ * [Actions](#actions)
+ * [Background](#background)
+ * [Goals](#goals)
+ * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+
+
+## Metadata
+
+
+|   |  |
+| -- | -- |
+| Software | https://github.com/linkerd/linkerd2/tree/main |
+| Security Provider | No |
+| Languages | Go, Rust, JavaScript, Shell |
+| SBOM | [Linkerd2 SBOM](https://github.com/amanda-gonzalez/tag-security/blob/b985fdda15ced11eeebb9e0cbd8e93a558719f17/assessments/projects/linkerd/linkerd2_linkerd_2d716299a1714728651f2ba7b08478b2f1e54605.json) |
+| | |
+
+
+### Security links
+
+
+Provide the list of links to existing security documentation for the project. You may
+use the table below as an example:
+| Doc | url |
+| -- | -- |
+| Security file | https://github.com/linkerd/linkerd2/blob/main/SECURITY.md |
+| Default and optional configs | N/A |
```

Comment:
Do you mean there isn't a consolidated documentation default and optional configs, or it is not applicable at all? I'm leaning toward the former, please confirm and update accordingly.

Reply:
this seems to be closest to default configs, although optional config options here https://github.com/linkerd/linkerd2/blob/main/BUILD.md#development-configurations, updated in doc